ccd-definition/CaseEvent/Camunda/CaseEvents.json

```diff
@@ -0,0 +1,41 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "CLAIM_ISSUE",
+    "Name": "Issue claim",
+    "Description": "Issue claim",
+    "PreConditionState(s)": "*",
+    "PostConditionState": "CASE_ISSUED",
+    "SecurityClassification": "Public",
+    "ShowSummary": "N",
+    "ShowEventNotes": "N",
+    "RetriesTimeoutURLAboutToSubmitEvent": 0
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "PAYMENT_FAILED",
+    "Name": "Payment failed",
+    "Description": "Payment failed",
+    "PreConditionState(s)": "*",
```

Comment:
We should be stricter here: pre condition state `PRENDING_CASE_ISSUED` same as post condition state